pallets/time-release/src/lib.rs

```diff
@@ -268,15 +323,10 @@ pub mod module {
 			let from = T::TransferOrigin::ensure_origin(origin)?;
 			let to = T::Lookup::lookup(dest)?;
 
-			if to == from {
-				ensure!(
-					T::Currency::balance(&from) >=
-						schedule.total_amount().ok_or(ArithmeticError::Overflow)?,
-					Error::<T>::InsufficientBalanceToFreeze,
-				);
-			}
+			let total_amount = schedule.total_amount().ok_or(ArithmeticError::Overflow)?;
+			Self::ensure_suffient_free_balance(&from, &to, total_amount)?;
```

Comment:
spelling for a used function
```suggestion
			Self::ensure_sufficient_free_balance(&from, &to, total_amount)?;
```